pkgs/development/python-modules/naked/default.nix

```diff
@@ -0,0 +1,57 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, unittestCheckHook
+, requests
+, nodejs
+, ruby
+, nix-update-script
+}:
+
+buildPythonPackage rec {
+  pname = "naked";
+
+  version = "0.1.32";
+
+  src = fetchFromGitHub {
+    owner = "chrissimpkins";
+    repo = "naked";
+    rev = "v${version}";
+    hash = "sha256-KhygnURFggvUTR9wwWtORtfQES8ANd5sIaCONvIhfRM=";
+  };
+
+  postPatch = ''
+  rm tests/test_NETWORK* -rf # no internet in sandbox
+
+  # TODO: disable only specific broken tests
+  rm tests/test_SYSTEM.py
+  rm tests/test_TYPES.py
+  rm tests/test_SHELL.py
```

Comment:
Can we use `pytestCheckHook` and `disabledTests`?

Reply:
It uses unittest instead of pytest

Reply:
pytest supports unittest. https://docs.pytest.org/en/7.4.x/#id1

Reply:
#TIL didn't know about that

Reply:
done, disabling those broken tests were a bit tricky but I got that